src/components/AddFacilityActionsPopover.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <ion-content>
+    <ion-list>
+      <ion-list-header>
+        {{ group.facilityGroupName }}
+      </ion-list-header>
+      <ion-item button @click="viewFacilities(group.facilityGroupId)">
+        {{ translate("View facilities") }}
+        <ion-icon :icon="openOutline" color="primary" slot="end" />
+      </ion-item>
+      <ion-item button lines="none" @click="openAddFacilityToGroupsModal(group.facilityGroupId)">
+        {{ translate("Quick edit") }}
+      </ion-item>
+    </ion-list>
+  </ion-content>
+</template>
+  
+<script lang="ts">
+import {
+  IonContent,
+  IonIcon,
+  IonItem,
+  IonList,
+  IonListHeader,
+  modalController,
+  popoverController
+} from "@ionic/vue";
+import { defineComponent } from "vue";
+import { translate } from '@hotwax/dxp-components'
+import { mapGetters, useStore } from "vuex";
+import { openOutline } from 'ionicons/icons'
+import AddFacilityToGroupsModal from '@/components/AddFacilityToGroupModal.vue'
+
+export default defineComponent({
+  name: "AddFacilityActionsPopover",
```

Comment:
Rename component to `GroupActionsPopover`, also update the related variable names.